qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -43,7 +43,7 @@ def test_scrub_abort(self):
         log.info("client_path: {0}".format(client_path))
 
         log.info("Cloning repo into place")
-        repo_path = TestScrubChecks.clone_repo(self.mount_a, client_path)
+        _ = TestScrubChecks.clone_repo(self.mount_a, client_path)
```

Comment:
just out of curiosity, why not just ignore the return value?

Reply:
Done

Reply:
there more occurrences of this pattern in this changeset. but i think it's fine.